gradle/scripts/scoverage.gradle

```
@@ -0,0 +1,13 @@
+/*
+ * Scala code coverage gradle plugin see https://github.com/scoverage/gradle-scoverage#configuration and
+ * https://github.com/scoverage/gradle-scoverage/issues/109 for details
+ */
+scoverage {
+	scoverageVersion = '1.4.1'
+	scoverageScalaVersion = '2.13.1'
+	coverageOutputHTML = false
+	coverageOutputXML = true
+	coverageOutputCobertura = false
+	minimumRate = 0.0 // minimum code coverage
```

your current coverage seems to be around 60%, so set this here might be a valid choice to prevent a reduction in coverage over time
```suggestion
	minimumRate = 0.6 // minimum code coverage
```